main/inc/lib/course_home.lib.php

```diff
@@ -909,6 +910,7 @@ public static function show_tools_category($all_tools_list)
                                     ICON_SIZE_SMALL,
                                     false
                                 );
+                                $link['name']='<em id="'.'linktool_'.$tool['iid'].'"class="fa fa-eye-slash text-muted" title="'.get_lang('Activate').'"></em>';
```

Comment:
Codacy found an issue: [Line exceeds maximum limit of 120 characters; contains 159 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5714830)